bin/pycbc_live

```diff
@@ -21,6 +23,19 @@ def makedir(path):
     if path is not None and not os.path.exists(path):
         os.makedirs(path)
 
+def ptof(p, ft):
+    return numpy.log(1.0-p) / -ft
+
+def ftop(f, ft):
+    return 1 - numpy.exp(-ft * f)
+
+def combine_ifar_pvalue(ifar, pvalue):
+    from scipy.stats import combine_pvalues
+    reference_foreground = 0.01 * ifar
```

Comment:
need comment on what this '0.01 * ifar' does and why (if we think so) it's ok and/or desirable